iree/samples/simple_embedding/simple_embedding.c

```diff
@@ -12,15 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// A example of setting up the HAL mddule to run simple pointwise array
-// multiplication with the dylib-sync driver.
+// A example of setting up the HAL module to run simple pointwise array
+// multiplication with the dylib driver.
```

Comment:
reword, this isn't just for the dylib driver now

Reply:
Done.